### PR TITLE
TX: Head Start Discovery

### PR DESCRIPTION
**Pull request by patwey**
## Program Details
- **Program**: Head Start
- **State**: TX
- **White Label**: TX
- **Research Date**: 2026-03-11

## Summary by CodeRabbit

* **New Features**
  * Added TX Head Start 2025 program configuration with metadata, descriptions, links, and required documents for application flow.
* **Tests**
  * Added extensive validation scenarios covering household compositions, income/FPL thresholds, age edge cases (ages ~3–5), duplicate-enrollment prevention, service-area checks, and multi-child situations.
* **Documentation**
  * Added a comprehensive program specification, human-review checklist, research references, and detailed test scenario descriptions.

### CODE REVIEW

File: validations/management/commands/import_validations/data/tx_head_start.json [hunk 1]
```diff
@@ -0,0 +1,1016 @@
...
+            "va": false
+          }
+        }
+      ],
+      "expenses": []
+    },
+    "expected_results": {
+      "program_name": "tx_head start",
+      "eligible": false
+    }
```
[the remark this answers is not in the document]
Comment (patwey) [reply validations/management/commands/import_validations/data/tx_head_start.json]: Good catch\! The notes say "Child Age 2" but the data shows `age=1, birth_year=2024` (child is ~22 months in March 2026). I'll update the notes to say "Child Age 1 (Below Minimum Age 3)" to match the actual data.

File: programs/programs/tx/head_start/spec.md [hunk 2]
```diff
@@ -0,0 +1,467 @@
...
+
+10. **Child must not yet be eligible for kindergarten** ⚠️ *data gap*
+   - Note: 5-year-olds are only eligible if not yet eligible for kindergarten. Kindergarten eligibility varies by state/district and depends on birth date cutoffs (e.g., must turn 5 by September 1). The screener captures age but not kindergarten enrollment status or eligibility.
+   - Source: 45 CFR § 1302.12(c)(1) - Age eligibility
+   - Impact: Medium
+
+11. **Residency in program service area** ⚠️ *data gap*
+   - Note: Each Head Start program serves a specific geographic area (may be county, city, neighborhood, or other boundary). The screener captures zipcode and county but cannot determine if these fall within a specific program's service area without program-specific geographic data.
+   - Source: 45 CFR § 1302.12(b) - Geographic eligibility
+   - Impact: High
```
[the remark this answers is not in the document]
Comment (patwey) [reply programs/programs/tx/head_start/spec.md]: Agreed — Scenario 9 can't actually test service-area eligibility since we don't have grantee boundary data. I'll reframe it as a standard income/age eligibility test for a Travis County resident (same as Scenario 1/3 but with explicit location context), removing any assertion that Travis County itself grants eligibility.

File: programs/management/commands/import_program_config_data/data/tx_head_start_initial_config.json [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+    "legal_status_required": [],
+    "name": "Head Start",
+    "description": "Head Start is a free program for young children ages 3 to 5. It offers preschool classes, meals, and health checkups. Your child also gets help learning and growing. Families pay nothing to join.\n\nFamilies with low income can apply. Children who get TANF (cash help) or SSI (disability benefits) qualify right away. Children in foster care also qualify. Children without a stable home qualify too.",
+    "description_short": "Free preschool for children ages 3 to 5",
+    "learn_more_link": "https://www.earlychildhood.texas.gov/about-eligibility-screener",
+    "apply_button_link": "https://www.earlychildhood.texas.gov/about-eligibility-screener",
+    "apply_button_description": "Apply for TX Head Start",
+    "estimated_application_time": "1 - 2 hours",
+    "estimated_delivery_time": "Varies by program availability",
+    "estimated_value": "Free preschool services valued at approximately $10,000 per year",
```
Comment (catonph) [inline programs/management/commands/import_program_config_data/data/tx_head_start_initial_config.json]: Let's leave blank and put in average childcare for this age group in texas or put "Varies"

**Review by catonph — CHANGES_REQUESTED** [review]
A few small-ish things

**Review by catonph — APPROVED** [review]
A few small things then good to go
